c/tskit/tables.c

```diff
@@ -4463,6 +4472,28 @@ cmp_edge(const void *a, const void *b)
     return ret;
 }
 
+static int
+cmp_migration(const void *a, const void *b)
+{
+    const migration_sort_t *ca = (const migration_sort_t *) a;
+    const migration_sort_t *cb = (const migration_sort_t *) b;
+
+    int ret = (ca->time > cb->time) - (ca->time < cb->time);
+    /* If time values are equal, sort by the source population */
```

Comment:
Perhaps worth noting the full sorting rules in `docs/data-model.rst`. Right now only the nondecreasing time value is listed.

Reply:
We decided to not put any further constraints on the input requirements. The extra keys are to make sure that the sort is always fully specified.